awscli/handlers.py

```diff
@@ -33,6 +33,7 @@
 from awscli.customizations.rds import register_rds_modify_split
 from awscli.customizations.putmetricdata import register_put_metric_data
 from awscli.customizations.sessendemail import register_ses_send_email
+from awscli.customizations.cloudfront import register as cloudfront_register
```

Comment:
We're not 100% consistent with this, but most of the register functions are imported as `register_<service>`.